cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/coreos/etcd/clientv3"
 	"github.com/coreos/etcd/pkg/transport"
 	"github.com/pkg/errors"
+	"go.etcd.io/etcd/etcdserver/api/v3rpc/rpctypes"
```

Comment:
i don't like the fact that we now have to include this package.
only few areas of k8s do that.

is there an alternative?

Reply:
@neolit123 i'm changing it to `"github.com/coreos/etcd/etcdserver/api/v3rpc/rpctypes"` for consistency with the other etcd imports but being able to check the error type directly is more reliable IMO than any alternative (like error string matching)